spec/clusterable/ht_item_spec.rb

```diff
@@ -82,25 +82,6 @@
     end
   end
 
-  xit "prevents duplicates from being created" do
```

Comment:
Mongo-specific and/or no longer relevant given that we aren't creating or updating htitems from holdings.